src/classes/particle.ts

```diff
@@ -151,11 +157,11 @@ export class Particle {
         return pos;
     }
 
-    private calcVelocity(options: IOptions) {
+    private calcVelocity(options: IOptions): IVelocity {
         const velbase = this.getVelBase(options);
-        let res: IVelocity = {
+        let res = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'res' is never reassigned; use 'const' instead of 'let'.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)